src/ui/buttons/props.js

```diff
@@ -190,6 +190,7 @@ export type RenderButtonProps = {|
     experiment : Experiment,
     vault : boolean,
     userIDToken : ?string,
+    applePay : Object,
```

Comment:
Can we use a full type for applePay? `Object` is very similar to using `any` and doesn't provide much type safety for inner keys.